### DISCUSSION TITLE
Wiley Journal Format Extension doesn't work anymore with latest Quarto version

### DISCUSSION THREAD

**Discussion by maxdrohde** [quarto-dev/quarto-cli#8796] [Q&A] [answered]
### Description

I upgraded to the latest Quarto version (1.4.550), and this Quarto extension that I had been using to create Wiley Journal articles stopped working.

https://github.com/ramiromagno/wiley-njd

I downgraded to Quarto 1.3.450 and it works on that. Whenever I try to render a document, I receive this error:

```
ERROR: Template partial partials/abstract.tex was not found. Please confirm that the path to the file is correct.

Stack trace:
    at file:///Applications/quarto/bin/quarto.js:41490:19
    at Array.flatMap (<anonymous>)
    at readPartials (file:///Applications/quarto/bin/quarto.js:41484:20)
    at runPandoc (file:///Applications/quarto/bin/quarto.js:68839:30)
    at eventLoopTick (ext:core/01_core.js:183:11)
    at async renderPandoc (file:///Applications/quarto/bin/quarto.js:70041:26)
    at async Object.onRender (file:///Applications/quarto/bin/quarto.js:77261:36)
    at async renderFileInternal (file:///Applications/quarto/bin/quarto.js:77238:17)
    at async renderFiles (file:///Applications/quarto/bin/quarto.js:77062:17)
    at async render (file:///Applications/quarto/bin/quarto.js:81807:21)
```

I'm not sure what changed in these later Quarto versions that would have caused this. Any advice would be greatly appreciated! 😄 

### Quarto Info

```
❯ quarto check
Quarto 1.4.550
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.4.550
      Path: /Applications/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: Installation From Path
      Path: /Library/TeX/texbin
      Version: 2023

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.11.3
      Path: /Library/Frameworks/Python.framework/Versions/3.11/bin/python3
      Jupyter: 5.3.0
      Kernels: ir, julia-1.6, julia-1.8, julia-1.9, juliapro_v1.5.0-1-1.5, julia-1.5, julia-12-threads-1.8, python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.3.2
      Path: /Library/Frameworks/R.framework/Resources
      LibPaths:
        - /Library/Frameworks/R.framework/Versions/4.3-x86_64/Resources/library
      knitr: 1.45
      rmarkdown: 2.25

[✓] Checking Knitr engine render......OK
```

**Comment by mcanouil** [comment, marked as answer]
You need to report that to the author(s).
The extension has undeclared dependencies to partials.
The error was silenced before but is no longer silenced.

When using an extension, first reach out to the extension authors on the repository where is the extension is.
Quarto team can eventually help the authors but that is after the authors have been made aware. Not the other way around as the number of extensions increases, Quarto team cannot debug the extension.

**Reply by maxdrohde** [reply]
Thank you @mcanouil. I have reported the issue to the extension developer. In the meantime, do you know if there is a way to silence the error in Quarto when rendering?

**Reply by mcanouil** [reply]
The extension is missing components.
So the extension has an issue.
You cannot really turn a blind eye on the error.

**Reply by maxdrohde** [reply]
I understand, thank you. I will try to work with the developer to fix the issue.

**Reply by mcanouil** [reply]
Not the error tells exactly what is missing.
The author tells Quarto about a partial, but it does not exist.
<img width="611" alt="Screenshot 2024-02-20 at 20 24 51" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/2f0c7082-0575-437f-b8e0-157b6e7c471c">

**Comment by cderv** [comment]
> I'm not sure what changed in these later Quarto versions that would have caused this.

In previous version of Quarto, we were silently ignoring the unfound file declaration. In newer Quarto, we are now throwing an error. 

So this was a hidden problem in the extension template. 

> In the meantime, do you know if there is a way to silence the error in Quarto when rendering?

When installed,e extension are installed in your folder project, so you can have access to the source file. You can then modify the `_extension.yml` file to remove the `template-partials` declaration that does not exists. 

This should remove the error. 

Hope it helps

**Reply by maxdrohde** [reply]
@cderv Thanks for the advice! I made a pull request with the developer to remove the reference to the missing partial, and everything is working now 😃
